### PR TITLE
Update README on how to enable check cache.

### PR DESCRIPTION
**Pull request by qiwzhang**
Add mixer filter config "check_cache_expiration_in_seconds" to support custom cache expiration.

### CODE REVIEW

File: src/envoy/mixer/README.md [hunk 1]
```diff
@@ -88,13 +89,20 @@ Notes:
 * quota_name, quota_amount are used for making quota call. quota_amount is default to 1 if missing.
```
Comment (geeknoid) [inline src/envoy/mixer/README.md]: quota_amount defaults to 1.

File: src/envoy/mixer/README.md [hunk 2]
```diff
@@ -88,13 +89,20 @@ Notes:
...
 * check_cache_keys is to cache check calls. If missing or empty, check calls are not cached.
 
-By default, mixer filter forwards attributes and does not invoke mixer server. You can customize this behavior per HTTP route by supplying an opaque config in the route config:
+## HTTP Route opaque config
+By default, mixer filter only forwards attributes and does not call mixer server. This behavior can be changed per HTTP route by supplying an opaque config:
```
Comment (geeknoid) [inline src/envoy/mixer/README.md]: the mixer

File: src/envoy/mixer/README.md [hunk 3]
```diff
@@ -111,3 +119,24 @@ Mixer client can be configured to make Quota call for all requests.  If "quota_n
 ## How to pass some attributes from client proxy to mixer.
 
 Usually client proxy is not configured to call mixer (it can be enabled in the route opaque_config). Client proxy can pass some attributes to mixer by using "forward_attributes" field.  Its attributes will be sent to the upstream proxy (the server proxy). If the server proxy is calling mixer, these attributes will be sent to the mixer.
+
+
+## How to enable cache for Check calls
+
+Check calls can be cached. By default, it is not enabled. It can be enabled by supplying non-empty "check_cache_keys" string list in the mixer filter config. Only these attributes in the Check request, their keys and values, are used to calculate the key for the cache lookup. If it is a cache hit, the cached response will be used.  The cached response will be expired in 5 minutes by default. It can be overrided by supplying "check_cache_expiration_in_seconds" in the mixer filter config.
```
Comment (geeknoid) [inline src/envoy/mixer/README.md]: I think it's worth mentioning that we intend to make this automatic such that it doesn't need to be configured by the operator.

File: src/envoy/mixer/README.md [hunk 4]
```diff
@@ -111,3 +119,24 @@ Mixer client can be configured to make Quota call for all requests.  If "quota_n
...
+
+Following is a sample mixer filter config to enable check cache:
```
Comment (geeknoid) [inline src/envoy/mixer/README.md]: enable the Check call caching.

File: src/envoy/mixer/README.md (hunk 1 above)
Comment (qiwzhang) [inline src/envoy/mixer/README.md]: Done

File: src/envoy/mixer/README.md (hunk 2 above)
Comment (qiwzhang) [inline src/envoy/mixer/README.md]: Done

File: src/envoy/mixer/README.md (hunk 3 above)
Comment (qiwzhang) [inline src/envoy/mixer/README.md]: Done

File: src/envoy/mixer/README.md (hunk 4 above)
Comment (qiwzhang) [inline src/envoy/mixer/README.md]: Done

**Pull request merged**